community/runc/APKBUILD

```diff
@@ -5,13 +5,13 @@ pkgname=runc
 pkgdesc="CLI tool for spawning and running containers according to the OCI specification"
 url="https://www.opencontainers.org"
 
-_commit=3e425f80a8c931f88e6d94a8c831b9d5aa481657
-pkgver=1.0.0_rc8
-pkgrel=2
+_commit=d736ef14f0288d6993a1845745d6756cfc9ddd5a
+pkgver=1.0.0_rc9
+pkgrel=0
 
-#_ver=v${pkgver/_rc/-rc}
+_ver=v${pkgver/_rc/-rc}
 # if we're building against an explicit commit beyond pkgver, use this instead:
-_ver=${_commit}
+#_ver=${_commit}
```

Comment:
why that changed?

Reply:
Prior to 1.0.0_rc9, we were building from a commit *after* 1.0.0_rc8's, and needed to pull the commit tarball.

With 1.0.0_rc9, we're back to pulling the version tarball, and so we've toggled back to that.